addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -2246,8 +2252,17 @@ const ListUserValueWidget = UserValueWidget.extend({
     /**
      * @private
      */
-    _onListItemChange() {
-        this._notifyCurrentState();
+    _onListItemBlurInput(ev) {
```

Comment:
As you added the "ev" param, it sould be added in the documentation
` * @param {Event} ev`